### DISCUSSION TITLE
Reopen an archived task

### DISCUSSION THREAD

**Discussion by fmatsos** [super-productivity/super-productivity#2522] [Q&A] [answered]
Hi !

I'm a fresh new user of Super Productivity and love it for the moment.
So I don't know all features completely yet and I'm wonder if is it possible to reopen a previous closed (Gitlab) task?

For example, I marked a task yesterday as done. But I had some feedbacks on it today and I would like to re-add the task (Gitlab issue) to my today board, for today's time tracking. But when I go on task, indicated as archived, I fall on month's summary.

Thank you!

**Comment by fmatsos** [comment, marked as answer]
OK i found how to do it. In month's summary you can restore task:
![image](https://user-images.githubusercontent.com/805227/225931964-404b4bd1-07be-4729-88da-85c7a378cc19.png)

I didn't notice the icon. Maybe add this feature in search dropdown too can improve UX :)

Sorry for the inconvenience!

**Reply by johannesjo** [reply]
That's an excellent suggestion! Thank you!!
